test/exec.js

```diff
@@ -79,6 +80,36 @@ assert.equal(shell.error(), null);
 assert.equal(result.code, 0);
 assert.equal(result.stdout, "'+'_'+'\n");
 
+// bash by default on unix
+if (common.platform !== 'win') {
+  var result = shell.exec('echo $0');
+  assert.equal(shell.error(), null);
+  if (shell.which('bash')) {
+    // node v0.10 and earlier don't seem to support the 'shell' option
```

Comment:
I feel like this is a bad idea, if we can't support bash everywhere, we shouldn't support it anywhere.